src/components/errorbars/plot.js

```diff
@@ -13,16 +13,13 @@ var d3 = require('d3');
 var isNumeric = require('fast-isnumeric');
 
 var subTypes = require('../../traces/scatter/subtypes');
-var styleError = require('./style');
-
 
 module.exports = function plot(traces, plotinfo, transitionConfig) {
     var isNew;
     var xa = plotinfo.x(),
         ya = plotinfo.y();
 
-    transitionConfig = transitionConfig || {};
-    var hasAnimation = isNumeric(transitionConfig.duration) && transitionConfig.duration > 0;
+    var hasAnimation = transitionConfig && transitionConfig.duration > 0;
```

Comment:
Hmm… I *think* you're correct. Trying to think of edge cases for === 0. The subtlety is that if you apply a transition, the config gets passed all the way down. *But* if `duration === 0`, it's an attr update instead of a transition. Transition happens at the next tick at the earliest, so it may be somewhat slower if you're passing updates through as fast as possible. This is basically the simulation case where you want to use the slightly-streamlined machinery of transitions but without actually triggering next-tick d3 transitions.